relayer/src/keyring.rs

```diff
@@ -282,14 +318,18 @@ impl KeyRing {
 }
 
 /// Decode an extended private key from a mnemonic
-fn private_key_from_mnemonic(mnemonic_words: &str) -> Result<ExtendedPrivKey, Error> {
+fn private_key_from_mnemonic(
+    mnemonic_words: &str,
+    coin_type: CoinType,
+) -> Result<ExtendedPrivKey, Error> {
     let mnemonic = Mnemonic::from_phrase(mnemonic_words, Language::English)
         .map_err(|e| Kind::InvalidMnemonic.context(e))?;
 
     let seed = Seed::new(&mnemonic, "");
 
     // Get Private Key from seed and standard derivation path
-    let hd_path = StandardHDPath::try_from("m/44'/118'/0'/0/0").unwrap();
+    let hd_path =
+        StandardHDPath::try_from(format!("m/44'/{}'/0'/0/0", coin_type.num()).as_str()).unwrap();
```

Comment:
It's better to hand the error here, instead of using unwrap. Since this value is based on user input it opens up for errors.